packages/story-editor/src/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -125,10 +146,6 @@ function PageAttachmentPanel() {
     [updateCurrentPageProperties, pageAttachment]
   );
 
-  const debouncedUpdate = useDebouncedCallback((props) => {
-    updatePageAttachment(props);
-  }, 300);
-
   const [isInvalidUrl, setIsInvalidUrl] = useState(
     !isValidUrl(withProtocol(url || '').trim())
   );
```

Comment:
```suggestion
  const [isInvalidUrl, setIsInvalidUrl] = useState(
    url && !isValidUrl(withProtocol(url || '').trim())
  );
```
Something like this, otherwise I have an error from the beginning (empty string is not considered valid by this library).
The same can be fixed in `link.js`

![image](https://user-images.githubusercontent.com/21036224/133602349-d7a116e8-3059-43d4-aafb-b325f55d070a.png)

Reply:
Ah, I thought I already fixed this :)

Thanks! Will adjust.